xc/common/utils/prjxray_create_place_constraints.py

```diff
@@ -189,26 +199,39 @@ def get_tile_dict(self):
 
 
 class ClockPlacer(object):
-    def __init__(self, vpr_grid, io_locs, blif_data, roi):
+    def __init__(self, vpr_grid, io_locs, blif_data, args):
```

Comment:
Rather than pass args directly, please pass roi and clock limit explicitly.  We want to avoid tying ClockPlacer directly to argparse.

Reply:
Fixed